client/components/payment-method-logos/index.tsx

```diff
@@ -0,0 +1,113 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { Fragment } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import {
+	Visa,
+	MasterCard,
+	Amex,
+	WooPay,
+	ApplePay,
+	GooglePay,
+	Discover,
+	JCB,
+	Afterpay,
+	Affirm,
+	Klarna,
+} from './Icons';
+
+import './style.scss';
+
+const Payments = [
```

Comment:
```suggestion
const PaymentMethods = [
```

Reply:
changed in [af009a5](https://github.com/Automattic/woocommerce-payments/pull/8977/commits/af009a5035144213ca7d93bc162bb035dd0a639d) . thx